File: src java/Ameba.java
public class Ameba { //Амеба каждые 3 часа делится
    public static void main(String[] args) {
        int a = 1;
        int hour = 0;
        int itog = 0;
        while (hour <= 24) {
            if (hour % 3 == 0) {
                System.out.println(hour + " ч. - " + a + " шт");
                itog = a;
                a = a * 2;
            }
            hour++;
        }
        System.out.println("-------------");
        System.out.println("Итог: " + itog + " шт");
    }
}

File: src/Ameba.kt
//Амеба каждые 3 часа делится
class Ameba {
    fun amebaKotlin() {
        var a = 1
        var hour = 0
        var itog = 0
        while (hour <= 24) {
            if (hour % 3 == 0) {
                println("$hour ч. - $a шт")
                itog = a
                a *= 2
            }
            hour++
        }
        println("-------------")
        println("Итог: $itog шт")
    }
}